### ISSUE TITLE
Toy Trainer Expects Separate Dataloaders But Gets Dictionary

### ISSUE THREAD

**Issue by christoph-blessing** [sinzlab/nnfabrik#66]
Currently the toy trainer expects a separate dataloader for the training, validation and test set. This interface is no longer compatible with the framework.
